src/constants.js

```diff
@@ -0,0 +1,35 @@
+export const UI = Object.freeze({
+  INPUT_QUERY: '덧셈할 문자열을 입력해 주세요.\n',
+  output: result => `결과 : ${result}`,
```

Comment:
```suggestion
  output: result => `결과 : ${result}`, // 상수일까? 유틸일까? 🤔
```
저도 예전에 이렇게 동적으로 값을 받아서 사용했던 경험이 있었는데, 상수라는 개념이 변하지 않는 값을 모아뒀는데 이렇게 동적으로 받아서 사용하면 사실상 유틸쪽이 가깝지 않나 고민이 되었고 최종적으로 유틸쪽에 더 가깝다고 판단하게 되었는데 이 부분에 대해서 예찬님의 의견이 궁금합니다!!

Reply:
저도 작성하고 제출하고 나서 이 부분에 대해서 고민이 들더라구요! 저도 지금의 형태는 유틸성이 강한것 같긴합니다!:)